### PR TITLE
Change quantization levels for SymmetricQuantizer from 255 to 256

### PR DESCRIPTION
**Pull request by aleksei-kashapov**
Quantization levels are changed for weights only as activations are already use full range of 8 bits

### CODE REVIEW

File: nncf/quantization/layers.py [hunk 1]
```diff
@@ -257,8 +257,6 @@ def calculate_level_ranges(num_bits, signed, is_weights):
             level_high = 2 ** num_bits - 1
             level_low = 0
         levels = 2 ** num_bits
-        if is_weights:
```
Comment (ljaljushkin) [inline nncf/quantization/layers.py:260]: It's strange that pre-commit is OK, while reference implementation in tests still has 255 levels. 

At least it should be also corrected in `test_functions.py`

Comment (aleksei-kashapov) [reply nncf/quantization/layers.py:260]: I suppose the tests were passed as in **test_functions.py** we utilize the same levels-255 for reference and symmetric_quantize.

**Comment by AlexKoff88** [comment]
@aleksei-kashapov, please report the status of E2E validation here as well.

File: tests/quantization/test_functions.py [hunk 2]
```diff
@@ -182,19 +182,15 @@ def calc_scale(input_):
                 return scales
 
         @staticmethod
-        def get_range_level(is_signed, is_weights, bits):
```
Comment (AlexKoff88) [inline tests/quantization/test_functions.py:185]: We will need this parameter when we introduce changes for the second part of the task that assumes having a workaround for the saturation issue.

**Review by vshampor — CHANGES_REQUESTED** [review]

**Review by vshampor — APPROVED** [review]

**Pull request merged**